tfidf-benchmark/NLP-pipeline-sklearn.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def performance_numbers(parquet_path, worker_counts=[1], samples=1, data_frac=0.2, result_path="./results.pkl", 
</pre><p>Remove <code>data_frac</code> as that's no longer used to ensure we have similar results. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>